text/103-leaving-x-twitter.md

```diff
@@ -0,0 +1,129 @@
+[mediapart]: https://www.mediapart.fr/journal/france/171224/contre-la-desinformation-mediapart-quitte-x
+[amu]: https://www.univ-amu.fr/en/public/actualites/aix-marseille-universite-suspends-its-activity-x
+[unistra]: https://www.unistra.fr/communiques-presse/detail-communique-de-presse-archive/22529-the-university-of-strasbourg-leaves-the-social-network-x-ex-twitter
+[unilux]: https://www.uni.lu/en/news/university-suspends-its-x-twitter-account/
+[rwth]: https://x.com/RWTH/status/1861320302124253564
+[uu]: https://www.uu.nl/en/news/utrecht-university-quits-x
+[york]: https://www.york.ac.uk/library/about/news/2024/ex-x/
+
+<summary>The arguments given by these universities to justify their departure from X/Twitter vary quite a lot (from the lack of respect for the European code of good conduct and the proliferation of false information and conspiracy theories to the diminishing reach of X/Twitter compared to other platforms):
+
+<details>
+
+>By withdrawing from the European code of good practice against misinformation and
+modifying its moderation rules, X has become a place where fake news, hateful, illicit or
+violent content is propagated. This social network is no longer aligned with our mission of
+transmitting knowledge and science, openness to others and tolerance.
+
+>This decision has been motivated by the evolution of this network: lack of respect for the European code of good conduct, paid certification, proliferation of false information and conspiracy.
+
+>The editorial and commercial evolution of X/Twitter in recent months has come to conflict with the University’s values of diversity, inclusion and social dialogue. The disbanding of the moderation team at X/Twitter and the fact that the social media platform pulled out from the European Union’s voluntary code to fight disinformation further informed this decision.
+
+>Due to the promotion of anti-democratic, anti-scientific content and conspiracy theories on the platform X, RWTH Aachen University is going to discontinue its activities on it. The platform is not sufficiently addressing hate speech, hatred against minorities, conspiracy theories, and right-wing extremist opinions – in short, it is not doing enough to combat radicalization. We will continue to provide information on our other social media channels (Instagram, LinkedIn, YouTube, TikTok, Threads and Facebook).
+
+>In recent years, UU's reach on X has decreased compared to other platforms, such as LinkedIn. In addition, we have seen how X, often more than other platforms, struggles with the rise of disinformation, fake accounts and online harassment.
+
+>This is for a number of reasons: primarily it no longer works as an effective method of distributing information to our communities. 
+</details>
+</summary>
+
+Also, six months after leaving X/Twitter, [NPR claims that the effect on their audience has been negligible](https://pluralistic.net/2023/10/14/freedom-of-reach/).
```

Comment:
Indeed, this source may not be the best. I also did not bother doing stats of what the CoqLang account brings (it would be actually very hard to know, because contrary to a media, what we are interested in is not web traffic).

>Have there been problems with misinformation, etc specific to CoqLang?

Not that I know of.